content/efcore/getting-started/querying.md

```diff
@@ -0,0 +1,47 @@
+# Querying Data
+
+Sometimes you really need an item (or items) from your database based off a certain set of parameters. In case you've forgotten, this is what querying is.
+
+Querying with EF Core uses LINQ statements (if you need some more pointers, check [this](https://www.microsoft.com/net/tutorials/csharp/getting-started/linq) out). For those who really like LINQ, [101 LINQ Samples](https://code.msdn.microsoft.com/101-LINQ-Samples-3fb9811b) is a great resource.
+
+The basic format of queries is:
+
+```c#
+variableToStoreQueryResult = context.Model.QueryKeyWord(m => m.Identifier == value)
+```
+
+## Loading All Data
+
+Grabs the entirety of a table.
```

Comment:
Normally I am all for using as few words as possible, but this seems too terse. At this point, it is no different than the docs, in which case we should just be adding interactivity to the docs. Do you have any ideas for how to increase the content? To be clear, I don't want to add fluff; it should still be useful information.